R/inherit-param-estimates.R

```diff
@@ -0,0 +1,123 @@
+
+#' Supported parameter estimates to inherit from parent model
+BBR_ESTIMATES_INHERIT <- c("theta", "sigma", "omega")
```

Comment:
Can you move this constant to `aaa.R` with the other constants?

Also, remove the `'` from the comment, because this is causing it render an `.Rd` file, which we don't need. After you remove that, please run `devtools::document()` and you should see that `.Rd` file disappear.